src/main/java/com/points/calculate/CalculatorTree.java

```diff
@@ -0,0 +1,100 @@
+package com.points.calculate;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.points.node.twochild.Node;
+
+
+public class CalculatorTree {
+	
+	private Node topNode;
+	
+	public CalculatorTree() {
+		topNode = null;
+	}
+	
+	public CalculatorTree(String equation) {
+		if (StringUtils.isEmpty(equation) ) {
+			topNode = null;
+		} else {
+			topNode = createCalculatorTreeFromString(equation);
+		}		
+	}
+	
+	private static Node createCalculatorTreeFromString(String equation) {	
```

Comment:
Seems you should be setting the head Node in non-static context. This is working like an init from the constructor.